routes/api/meetup.js

```diff
@@ -7,10 +7,10 @@ var request = require('request');
 
 exports.meetups = function (req, res) {
     'use strict';
-    var url = 'http://api.meetup.com/2/events?status=upcoming&order=time&limited_events=False&group_urlname=node-dc&desc=false&member_id=26710252&offset=0&format=json&page=20&fields=&sig_id=26710252&sig=f9a8c7e4076e5ed5825f06378a855415af3722a9';
+    var url = 'https://api.meetup.com/2/events?offset=0&format=json&limited_events=False&group_urlname=node-dc&photo-host=public&page=20&fields=&order=time&desc=false&status=upcoming&sig_id=26710252&sig=dc51b92626c66cdedabf20d5c39aceb4720eb70a';
```

Comment:
This is what a signed API url is:

> Similar to OAuth signing, this method of authentication certifies that a request was approved by a particular user. Unlike OAuth-signed requests, key-signed requests may be reused and recycled as long as their corresponding API key is valid. If a signed URL is released to the public, any application can use it to interact with Meetup as if it had that API key; the difference is that it can not change definitive parameters or use the signature against other API methods.

We can use an API key as well, so maybe we should move it to an environment variable. Thoughts?